Add note about using sudo for the deb package (#208)
thelounge/thelounge.github.io: _docs/install-and-upgrade.md

The Lounge is now up and running **in private mode** at <http://localhost:9000>.

Its configuration file is located at `/etc/thelounge/config.js`. 

> To learn how to configure The Lounge, go to [the configuration section](/docs/configuration). To learn how to add users, read [the users section](/docs/users). Keep in mind that when using the `thelounge` cli command, you need to run it as the `thelounge` user, e.g `sudo -u thelounge thelounge --help`.

Reviewer: ```suggestion
To learn how to configure The Lounge, go to [the configuration section](/docs/configuration). To learn how to add users, read [the users section](/docs/users). The `thelounge` command needs to be run as the `thelounge` user, e.g. `sudo -u thelounge thelounge --help`.
```
